loadDataIntoAstraDB/README.md

```diff
@@ -8,14 +8,16 @@ In this guide, we'll explore multiple ways to load data into an Astra database.
 - Verify both datasets using the CQL Console
 
 # Prerequisites
-While not required for _all_ of the following sections you'll want to have 
+While not required for _all_ the following sections, you'll want to have: 
 - Access to a local terminal
 - Java 8 or higher
 
 ## 1 Create a database to explore data loading
-While you don't _have_ to use the following database and keyspace name, it would be best if you do to follow the examples below. If not, you'll have to replace with your own values.
+While you don't _have_ to use the following database and keyspace name, it would be best if you do to follow the examples below. If not, you'll have to replace them with your own values.
```

Comment:
```suggestion
You don't _have_ to use the following database and keyspace name; it would be best if you do to follow the examples below. If not, you have to replace them with your own values.
```